.github/workflows/macos.yml

```diff
@@ -0,0 +1,77 @@
+name: CI macOS

```

Comment:
Can we null this name and simplify what is displayed per job? in the github list this name prefixes all the sub-names which seems pointless.

Reply:
Good idea, will change that with the next iteration.

Reply:
Unfortunately it seems like it is not possible to completly nullify/hide it. If I set it to an empty string, GitHub will instead show the path to the YAML workflow file:
![image](https://user-images.githubusercontent.com/231943/77061675-ea206500-69da-11ea-9387-ca3117cce1d2.png)

I suggest to shorten the name to "CI" if it does not need to be unique inside the repository.

Reply:
Okay, so it seems like my suggestion works good and this is how it will look:
![image](https://user-images.githubusercontent.com/231943/77062152-9f531d00-69db-11ea-9275-a2b8346a8425.png)
@bagder What do you think? Is this sufficient?

Reply:
Oh yes, that's way better!